.circleci/Dockerfile

```diff
@@ -66,3 +66,4 @@ RUN cd /tmp && \
     sudo mv linux-amd64/helm /usr/local/bin && \
     rm -rf helm.tgz linux-amd64
 
+RUN wget https://storage.cloud.google.com/istio-tools/push_gcs_gubernator/push_gcs_gubernator
```

Comment:
How big is it ? Is it a script ? Can we check it in ?

Reply:
Yeah I removed this from docker image and create a separate scripts in bin/. See https://github.com/istio/test-infra/pull/769 for the source of ci_to_gubernator